resources/css/combined.css

```diff
@@ -7,19 +7,19 @@ body{
 	margin: 0 auto;
 	padding-top: 90px;
 }
-.row{
-	margin: 0;
-}
 .full-jumbotron{
 	width:100%;
 }
 .jumbotron{
-	max-width:1444px;
-	margin: 0 auto;
+	
 	background:white;
 
 	border-bottom:1px solid #f1f1f1;
-	padding:70px 0;
+	
```

Comment:
Some weird spacing in this .jumbotron class (realize a lot of this was already here): should remove extra lines, and needs space between colons and values